crystal_helper.py

```diff
@@ -0,0 +1,229 @@
+import numpy as np
+import sys
+import re
+from pymatgen.symmetry.analyzer import SpacegroupAnalyzer
+from pymatgen.core.structure import Structure
+import itertools
+import pickle
+
+with open(r"bs_dict.pkl", "rb") as bs_dict_file:
+    basis_sets = pickle.load(bs_dict_file)
+    
+currently_used_bs=['H_pob_DZVP_rev2','', 'Li_pob_DZVP_rev2', 'Be_pob_DZVP_rev2', 'B_pob_DZVP_rev2', 'C_pob_DZVP_rev2', 'N_pob_DZVP_rev2', 'O_pob_DZVP_rev2', 'F_pob_DZVP_rev2','Ne_6-31G', 'Na_pob_DZVP_rev2', 'Mg_pob_DZVP_rev2', 'Al_pob_DZVP_rev2', 'Si_pob_DZVP_rev2', 'P_pob_DZVP_rev2', 'S_pob_DZVP_rev2', 'Cl_pob_DZVP_rev2', 'Ar_6-31G','K_pob_DZVP_rev2', 'Ca_pob_DZVP_rev2', 'Sc_pob_DZVP_rev2', 'Ti_pob_DZVP_rev2', 'V_pob_DZVP_rev2', 'Cr_pob_DZVP_rev2', 'Mn_pob_DZVP_rev2', 'Fe_pob_DZVP_rev2', 'Co_pob_DZVP_rev2', 'Ni_pob_DZVP_rev2', 'Cu_pob_DZVP_rev2', 'Zn_pob_DZVP_rev2', 'Ga_pob_DZVP_rev2', 'Ge_pob_DZVP_rev2', 'As_pob_DZVP_rev2', 'Se_pob_DZVP_rev2', 'Br_pob_DZVP_rev2','', 'Rb_POB_DZVP_2018', 'Sr_POB_DZVP_2018','Y_POB_DZVP_2018', 'Zr_POB_DZVP_2018', 'Nb_POB_DZVP_2018','','Mo_POB_DZVP_2018', 'Ru_POB_DZVP_2018', 'Rh_POB_DZVP_2018', 'Pd_POB_DZVP_2018', 'Ag_POB_DZVP_2018', 'Cd_POB_DZVP_2018', 'In_POB_DZVP_2018', 'Sn_POB_DZVP_2018', 'Sb_POB_DZVP_2018', 'Te_POB_DZVP_2018', 'I_POB_DZVP_2018']
+    
+def find_elements(structure):
+    ineq_elements=list(set(structure.species))
+    ineq_Z=[elem.Z for elem in ineq_elements]
+    return ineq_Z
+    
+class Crystal_input:
+    
+    def __init__(self,input_name,structure,FORCECALC=False):
+        self.name = input_name
+        self.structure = structure
+        self.elements = find_elements(self.structure)
+        self.shrink = self.cry_shrink()
+        self.FORCECALC = FORCECALC
+        
+        
+    def write_cry_gui(self,gui_file,dimensionality=3):
+
+        with open(gui_file, 'w') as file: 
+
+            atoms = SpacegroupAnalyzer(self.structure).get_primitive_standard_structure()
```

Comment:
Please use this updated version of this function:
https://github.com/bcamino/crystal_functions/blob/main/crystal_functions/file_readwrite.py
line 788

 We need to take into account the no symmetry cases